test/strict.coffee

```diff
@@ -17,8 +17,10 @@
 # * `eval` or `arguments` as the operand of a post/pre-fix inc/dec-rement expression
 
 # helper to assert that code complies with strict prohibitions
-strict = (code, msg) ->
+strict = (code, msg, {ast} = {}) ->
```

Comment:
...same idea here

Reply:
The idea is fine, but I’m not a fan of mixing styles. We can refactor to move to options arguments later:

```coffee
strict = (code, msg, ast = no) ->
```